src/main/java/com/keshavarzi/airplanestats/model/RouteEntity.java

```diff
@@ -0,0 +1,63 @@
+package com.keshavarzi.airplanestats.model;
+
+import jakarta.annotation.Nonnull;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.ForeignKey;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "route", schema = "user_data")
+public class RouteEntity {
```

Comment:
Unclear on what a `RouteEntity` is, suggest adding a class level java doc. What should it encapsulate? A whole route? How will it handle multi-stop flights? If it is only one segment, then I have a nit to change this to `SegmentEntity`?

Reply:
It is supposed to be a "segment", a `Trip` can encompass zero, one, or more flight routes.  Multiple connections can be added manually to a `Trip`.  If that makes sense to you, it can be resolved

Reply:
Resolved per our conversation